doc/source/getting_started/notebooks/notebook2python.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
If we decide to let Travis take care of the Python script generation, bash is fine.